src/ui/public/courier/fetch/request/__tests__/segmentedCreateQueue.js

```diff
@@ -0,0 +1,62 @@
+describe('ui/courier/fetch/request/segmented/_createQueue', () => {
```

Comment:
Why not just add all of this as a describe block in the existing `segmented` test?